CONTRIBUTING.md

```diff
@@ -71,30 +104,19 @@ branch:
     mkdir tutorials/Spectral-Line-Fitting
 
 All files used by the tutorial -- e.g., example data files, the IPython
-notebook file itself -- should go in this directory. Now you can start writing
-the tutorial! Change directories into this new path and start up an
-IPython notebook server:
-
-    cd tutorials/Spectral-Line-Fitting
-    ipython notebook --matplotlib inline
-
-Create a new notebook file, and write away! (Following the Content Guidelines above.)
-Remember to place any extra files
-used by the tutorial in the directory with the notebook file, and place them
-under git version control.
+notebook file itself -- should go in this directory. 
 
 You will also need to edit the notebook file metadata.
 (IPython notebook --> edit menu --> edit notebook metadata)
-The metadata contains any extra information about the tutorial you may want to add.
 The metadata must contain, at minimum, the following fields:
 
 - link_name (the name of the link which will appear in the list of tutorials)
-- author
+- author list
 - date (month year, e.g. 'July 2013')
 
 Here is an example of one of these files: [FITS-header.ipynb](https://github.com/astropy/astropy-tutorials/blob/master/tutorials/FITS-header/FITS-header.ipynb) (be sure to hit the "raw" button to see the metadata).
 
-You will also need to specify any python packages that the tutorial depends on.
+Please specify any python packages that the tutorial depends on via the `requirements.json` file.
```

Comment:
This will change as well, but I'm fine leaving this & merging and then I can rebase my PR (#153) and update with the new info.